docs/docs/tools/Shopify Order Status.md

```diff
@@ -0,0 +1,41 @@
+---
+sidebar_position: 3
+---
+
+This tool allows chatbots to check the status of orders in your Shopify store.
+
+Your customers can specify their email and order number and the chatbot will return the status of the order.
+
+## Functionality
+
+- Check order status
+- Escalate the conversation if order cannot be found or there is an issue with the order
+
+## Setup
+
+### Install Algomo app in your Shopify Store
+
+1. [https://apps.shopify.com/algomo](https://apps.shopify.com/algomo)
+2. …
+
+### Create a tool
+
+1. Create a tool
+
+### Activate the tool
+
+1. Assign tool to a chatbot
+2. Assign chatbot to a widget
+3. Test the widget
+
```

Comment:
I think a section here about how to test the tool would be great

### Test the tool
you can ... by ...
For example... "Where is my order"